src/main.rs

```diff
@@ -140,7 +140,7 @@ async fn main() {
             |job| async {
                 ObsJobHandler::from_obs_config_in_job(job, HandlerOptions::default()).map_err(
                     |err| {
-                        error!("Failed to create new client: {:?}", err);
+                        error!(gitlab.output = true, "Failed to handle new job: {:?}", err);
```

Comment:
gitlab.output is really an implementation detail that crates using the runner shouldn't be aware of :). If you use `outputln!` it will already do the right thing; If you want and equivalent of e.g. `error!` then that's something to look at in the gitlab runner crate really.